pkg/destroy/openstack/openstack.go

```diff
@@ -1274,6 +1278,102 @@ func deleteSnapshots(conn *gophercloud.ServiceClient, volumeID string, logger lo
 	return true, nil
 }
 
+func deleteShares(opts *clientconfig.ClientOpts, filter Filter, logger logrus.FieldLogger) (bool, error) {
+	logger.Debug("Deleting OpenStack shares")
+	defer logger.Debugf("Exiting deleting OpenStack shares")
+
+	var clusterID string
+	for k, v := range filter {
+		if strings.ToLower(k) == "openshiftclusterid" {
+			clusterID = v
+			break
+		}
+	}
+
+	conn, err := clientconfig.NewServiceClient("sharev2", opts)
+	if err != nil {
+		logger.Error(err)
+		return false, nil
```

Comment:
This causes the destroy code to loop over `deleteShares()` when Manila is not available in the target cloud. We can't assume it's available since Manila is not a core OpenStack service.